include/odp/api/spec/ipsec.h

```diff
@@ -147,6 +147,25 @@ typedef struct odp_ipsec_inbound_config_t {
 	 */
 	odp_proto_chksums_t chksums;
 
+	/** Post-IPSEC reassembly configuration
+	 *
+	 *  When enabled, reassembly of inbound IPSEC processed packets would be
+	 *  attempted. This will allow reassembly of pre-IPSEC fragmented
+	 *  packets.
+	 *
+	 *  Reassembly cannot be attempted with SYNC mode of operation. For
+	 *  inline and asynchronous mode of operation, reassembly would be
+	 *  attempted for eligible packets.
+	 *
+	 *  When reassembly is attempted, stats would reflect the number
+	 *  of IPSEC processed packets, and so would count all individual
+	 *  fragments.
+	 *
+	 *  Reassembly will be attempted only if retain_outer is
+	 *  ODP_PROTO_LAYER_NONE.
+	 */
+	odp_reassembly_config_t reassembly;
+
 } odp_ipsec_inbound_config_t;
```

Comment:
Suggested rewording:

```
	/** Post-IPsec reassembly configuration
	 *
	 *  This field provides global IPsec configuration parameters for
	 *  fragment reassembly. The enable flag does not turn on reassembly
	 *  but tells if reassembly may be enabled in SA parameters.
	 *
	 *  The enable flag may be set only if retain_outer is ODP_PROTO_LAYER_NONE.
	 */
```

...but I am not sure if it is ok to just disallow retain_outer like that.

Reply:
Agreed.